src/mixins/planningActionMixin.js

```diff
@@ -241,18 +241,25 @@ export const planningActionMixin = {
         });
     },
     async onOkForEventCreation () {
-      this.loading = true;
-      const payload = this.getCreationPayload(this.newEvent);
-      if (!this.isCreationAllowed(payload)) {
-        return NotifyNegative('Impossible de créer l\'évènement : il est en conflit avec les évènements de l\'auxiliaire.');
-      }
+      try {
+        this.loading = true;
+        const payload = this.getCreationPayload(this.newEvent);
+        if (!this.isCreationAllowed(payload)) {
+          return NotifyNegative('Impossible de créer l\'évènement : il est en conflit avec les évènements de l\'auxiliaire.');
+        }
 
-      await this.$events.create(payload);
+        await this.$events.create(payload);
 
-      await this.refresh();
-      this.creationModal = false;
-      NotifyPositive('Évènement créé');
-      this.loading = false;
+        await this.refresh();
+        this.creationModal = false;
+        NotifyPositive('Évènement créé');
+        this.loading = false;
```

Comment:
tu peux utiliser un `finally` pour set `this.loading` car il doit être `false` si la requête a réussi ou échoué.